test/browser_tests/spec_suites/leadership-calendar.js

```diff
@@ -30,41 +30,68 @@ describe( 'The Leadership Calendar Page', function() {
     }
   );
 
-  it( 'should include a search filter',
-    function() {
-      expect( page.searchFilter.isPresent() ).toBe( true );
-    }
-  );
+  describe( 'Search Filter', function() {
 
-  it( 'should include a search filter button',
-    function() {
-      var searchFilterBtn = page.searchFilterBtn;
-      expect( searchFilterBtn.isPresent() ).toBe( true );
-      expect( searchFilterBtn.getText() ).toContain( 'Filter calendars' );
-      expect( searchFilterBtn.getText() ).toContain( 'Show' );
-    }
-  );
+    it( 'should be included',
+      function() {
+        expect( page.searchFilter.isPresent() ).toBe( true );
+      }
+    );
 
-  it( 'should include a search filter search filter results',
-    function() {
-      expect( page.searchFilterResults.count() ).toBeGreaterThan( 0 );
-    }
-  );
+    it( 'should include Filter Calendars label',
+      function() {
+        var searchFilterBtn = page.searchFilterBtn;
+        expect( searchFilterBtn.isPresent() ).toBe( true );
+        expect( searchFilterBtn.getText() ).toContain( 'Filter calendars' );
+      }
+    );
 
-  it( 'should include a download filter',
-    function() {
-      expect( page.paginationForm.isPresent() ).toBe( true );
-    }
-  );
+    it( 'should include a visible Show button', function() {
+      expect( page.searchFilterShowBtn.isDisplayed() ).toBe( true );
+      expect( page.searchFilterShowBtn.getText() ).toBe( 'Show' );
+    } );
 
-  it( 'should include a download filter button',
-    function() {
-      var downloadFilterBtn = page.downloadFilterBtn;
-      expect( downloadFilterBtn.isPresent() ).toBe( true );
-      expect( downloadFilterBtn.getText() ).toContain( 'Download options' );
-      expect( downloadFilterBtn.getText() ).toContain( 'Show' );
-    }
-  );
+    it( 'should include a visible Hide button when clicked', function() {
+      page.searchFilterBtn.click();
+      browser.sleep(1000);
```

Comment:
It would be preferable to avoid `browse.sleep` since it's potentially flaky and may slow tests. You can get around it and have the test wait the actual time needed with e.g.:

```
      var expectedConditions = protractor.ExpectedConditions;
      page.searchFilterBtn.click();
      browser.driver.wait(
        expectedConditions.elementToBeClickable( page.searchFilterHideBtn )
      ).then( function() {
        expect( page.searchFilterHideBtn.isDisplayed() ).toBe( true );
      } );
```

I think that should do it.